mods/ra/languages/pl.ftl

```diff
@@ -0,0 +1,21 @@
+## ServerListLogic
+
+players-online = {$players ->
+    [one] {$players} Gracz Online
+   *[other] {$players} Gracze Online

```

Comment:
In Polish there is a distinction between "few" (numbers ending on 2-4) and "many". According to the Fluent docs this should be covered by the rule [few]. So (if I'm not mistaken):
```
players-online = {$players ->
    [one] {$players} gracz Online
    [few] {$players} gracze Online
   *[many] {$players} graczy Online
```

Slavic languages provide many such cases, Slovenian for example has a dual number as well:
```
    [one] {$players} igralec Online
    [two] {$players} igralca Online
    [few] {$players} igralci Online
   *[other] {$players} igralcev Online
```

Reply:
Interesting. I asked a native Polish speaker to translate for me, but I forgot to tell him about the ftl syntax. Added your sl example as well.